docs/src/examples/spindle_fr.adoc

```diff
@@ -141,39 +149,61 @@ net spindle-ready <= spindle-at-speed.out => spindle.N.at-speed
 _Index codeur broche_ sont choisis parmi les entrées.
 
+The StepConf Wizard can perform the connections for a single-spindle
+configuration for you if you select Encoder Phase A and Encoder Index as
+inputs.
+
 Matériel supposé présent:
 
- - Un codeur est monté sur la broche et délivre 100 impulsions par tour
-   sur son canal A.
- - Ce canal A est raccordé à la broche 10 du port parallèle.
- - L'index de ce codeur est connecté à la broche 11 du port parallèle.
+* Un codeur est monté sur la broche et délivre 100 impulsions par tour
+  sur son canal A.
+* Ce canal A est raccordé à la broche 10 du port parallèle.
+* L'index de ce codeur est connecté à la broche 11 du port parallèle.
 
 Configuration de base pour ajouter ces composants:
+footnote:[In this example, we will assume that some encoders have already
+been issued to axes/joints 0, 1, and 2. So the next encoder available for us
+to attach to the spindle would be number 3. Your situation may differ.]
+footnote:[The HAL encoder index-enable is an exception to the rule in that
+it behaves as both an input and an output, see the
+<<sec:encoder,Encoder Section>> for details]
+footnote:[It is because we selected 'non-quadrature simple counting...' above
+that we can get away with 'quadrature' counting without having any
+B quadrature input.]
+
+----
+# add the encoder to HAL and attach it to threads.
+loadrt encoder num_chan=4
+addf encoder.update-counters base-thread
+addf encoder.capture-position servo-thread
+
+# set the HAL encoder to 100 pulses per revolution.
+setp encoder.3.position-scale 100
+
+# set the HAL encoder to non-quadrature simple counting using A only.
+setp encoder.3.counter-mode true
+
+# connect the HAL encoder outputs to LinuxCNC.
+net spindle-position encoder.3.position => spindle.0.revs
+net spindle-velocity encoder.3.velocity => spindle.0.speed-in
+net spindle-index-enable encoder.3.index-enable <=> spindle.0.index-enable
+
+# connect the HAL encoder inputs to the real encoder.
+net spindle-phase-a encoder.3.phase-A <= parport.0.pin-10-in
+net spindle-phase-b encoder.3.phase-B
+net spindle-index encoder.3.phase-Z <= parport.0.pin-11-in
 ----
-loadrt encoder num_chan=1 
-addf encoder.update-counters base-thread 
-addf encoder.capture-position servo-thread 
-setp encoder.0.position-scale 100 
-setp encoder.0.counter-mode 1 
-net spindle-position encoder.0.position => spindle.N.revs 
-net spindle-velocity encoder.0.velocity => spindle.N.speed-in 
-net spindle-index-enable encoder.0.index-enable <=> spindle.N.index-enable 
-net spindle-phase-a encoder.0.phase-A 
-net spindle-phase-b encoder.0.phase-B 
-net spindle-index encoder.0.phase-Z 
-net spindle-phase-a <= parport.0.pin-10-in 
-net spindle-index <= parport.0.pin-11-in
-----
-
-[[sec:Vitesse-Broche-Atteinte]]
-== Vitesse broche atteinte
-(((Vitesse broche atteilte)))
+
+=== Vitesse broche atteinte
+[[sec:Vitesse-Broche-Atteinte]](((Vitesse broche atteilte)))
```

Comment:
Likewise, shouldn't we agree how to format titles/headings and tags: same line vs split lines ?
```suggestion
=== Vitesse de broche atteinte
[[sec:vitesse-broche-atteinte]](((Vitesse de broche atteinte)))
```